src/connection.cc

```diff
@@ -76,24 +68,17 @@ bool Connection::handle_read(const boost::system::error_code& error,
   }
 
   // check if it was done or not
-
-  // detect if request_handler never got initialized
-  // if so set it to a default handler, or maybe just return false ?
-  Response response;
-  // request_handler->handle_request(request, response);
-
-    /* Also note you can process the handler specific config
-       into a nice map using parseHandlerConfig();
-       which is defined in server_info.h but is throwing compile error heh */
-    
-    do_write(response);
-
-  // if the request handler is not found, do_read
-  // }
-  // else {
-  //   do_read();
-  // }
-
+  if (request_handler != NULL) {
+    Response* response = new Response();
```

Comment:
memory leak! Somehow, we need to be able to delete it in `handle_write`. Maybe store a pointer to the response as a member variable?